### ISSUE TITLE
Open file passed as argument via CLI

### ISSUE THREAD

**Issue by LeoBorai** [whizzes/favicon#1]
Open file passed as argument via CLI as follows:

```bash
favicon ./logo.png
```

Then print it's height and width to the terminal.

```bash
favicon ./logo.png
height: 120px
width: 120px
```

**Comment by LeoBorai** [comment]
I believe we can use: https://docs.rs/image/latest/image/struct.ImageBuffer.html

**Comment by LeoBorai** [comment]
Resolved on 5d354b23a856698bb3e414b9750e10d7ecbe7aa2
